packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -85,7 +78,7 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
         const rateLimiter = new RateLimiter(this.driverPolicies.maxConcurrentOrdererRequests);
         const ordererRestWrapper = await RouterliciousOrdererRestWrapper.load(
             tenantId,
-            id,
+            undefined,
```

Comment:
Does this work for t9s and r11s?

Reply:
Yes, both have support for it after #7022 + #7038 (for R11s) and #7104 (for T9s)